docs/en/Running-Flink-on-Alluxio.md

```diff
@@ -45,15 +45,11 @@ Next, you have to specify the path to the Hadoop configuration in Flink. Open th
 configuration value to the **directory** containing the `core-site.xml`. (For newer Hadoop versions,
 the directory usually ends with `etc/hadoop`.)
 
-### Generate and Distribute the Alluxio Client Jar
+### Distribute the Alluxio Client Jar
 
 In order to communicate with Alluxio, we need to provide Flink programs with the Alluxio Core Client
 jar. If you are using Alluxio binary, then compilation is unnecessary and the jar is available at
```

Comment:
We can leave out "compilation is unnecessary" - Users of the distribution tarball wouldn't expect to need to compile

Reply:
Done.